frontend/app/forms/MemberForm.scala

```diff
@@ -204,7 +219,7 @@ object MemberForm {
   )(PaymentForm.apply)(PaymentForm.unapply)
 
   val feedbackMapping: Mapping[FeedbackForm] =   mapping(
```

Comment:
This variable (`feedbackMapping`) is unused, you can remove it, and save 5 lines!